docs/user-guide/projects/project-lists/README.md

```diff
@@ -94,14 +95,20 @@ Projects can be filtered in OpenProject. The default view will list all currentl
 
 **Archived projects** - returns all projects that are not active, of which you were a member or have the right to see.
 
+**Public project lists** - shows all project lists that are public, i.e. that were shared with everyone within an OpenProject instance.
```

Comment:
@MayaBerd this entry will not exist as soon as the bug **56444** gets fixed because all the shared lists independent if they are shared individually or with everyone will go bellow "Shared project lists".